src/Reducers/notesActions-reducer.jsx

```diff
@@ -1,21 +1,21 @@
 
 const notesReducer = (state, action) => {
-    
+
     switch (action.type) {
         case "ADD_NOTE":
-            return {...state, notes : action.payload };
+            return {...state, notes : action.payload, allNotesData : action.payload };
         
         case "DELETE_NOTE":
-            return {...state, notes : action.payload };
+            return {...state, notes : action.payload, allNotesData : action.payload };
 
         case "EDIT_NOTE":
-            return {...state, notes : action.payload };
+            return {...state, notes : action.payload, allNotesData : action.payload };
 
         case "ARCHIVE_NOTE":
-            return {...state, notes : action.payload.notes, archives : action.payload.archives };
+            return {...state, notes : action.payload.notes, archives : action.payload.archives, allNotesData : action.payload.notes };
 
         case "RESTOR_ARCHIVE_NOTE":
```

Comment:
if it is supposed to be Restore, then you can correct the spelling `RESTORE_ARCHIVE_NOTE, can avoid confusion.